package.json

```diff
@@ -31,6 +31,7 @@
     "howler": "^2.1.3",
     "keypress.js": "^2.1.5",
     "lodash": "4.17.21",
+    "simplify-traditional": "^1.0.0",
```

Comment:
move this to dev dep please. it's not required for runtime.

Reply:
Sure , I,m sorry

I will write a readme